src/gui/addnewtorrentdialog.cpp

```diff
@@ -58,17 +59,18 @@
 #include "ui_addnewtorrentdialog.h"
 #include "addnewtorrentdialog.h"
 
-#define SETTINGS_KEY(name) "AddNewTorrentDialog/" name
-const QString KEY_ENABLED = SETTINGS_KEY("Enabled");
-const QString KEY_DEFAULTCATEGORY = SETTINGS_KEY("DefaultCategory");
-const QString KEY_TREEHEADERSTATE = SETTINGS_KEY("TreeHeaderState");
-const QString KEY_WIDTH = SETTINGS_KEY("Width");
-const QString KEY_EXPANDED = SETTINGS_KEY("Expanded");
-const QString KEY_TOPLEVEL = SETTINGS_KEY("TopLevel");
-const QString KEY_SAVEPATHHISTORY = SETTINGS_KEY("SavePathHistory");
-
 namespace
 {
+    #define SETTINGS_KEY(name) "AddNewTorrentDialog/" name
```

Comment:
We don't indent preprocessor dorectives.

Reply:
directives

Reply:
Done.